lib/filters/global/actions.js

```diff
@@ -17,7 +17,12 @@ module.exports = {
                     body.format = 'json';
                     return restbase[req.method](req)
                     .then(function(res) {
-                        if (res.status !== 200) {
+                        if (
+                            res.status !== 200
```

Comment:
The action API wrapper is really a very embryonic internal thing right now. Once we start thinking about how we could integrate the PHP API into a more general REST API we'd probably want to give it a more systematic treatment.

In the meantime, internal uses that don't actually supply a proper query should probably fail loud & early, rather than just seeming to succeed without results. It did so far (by accessing something that wasn't defined), but we could make it more explicit with a check & an explicit throw.